packages/ag-charts-website/src/features/docs/utils/urlPaths.ts

```diff
@@ -10,7 +10,7 @@ export function getFrameworkFromPath(path: string) {
 }
 
 export const getExamplePageUrl = ({ framework, path }: { framework: Framework; path: string }) => {
-    return pathJoin(SITE_BASE_URL, framework, path);
+    return pathJoin(SITE_BASE_URL, framework, path) + '/';
```

Comment:
In future, we'll want to fix `pathJoin` to support this scenario like in https://github.com/ag-grid/ag-charts/pull/534/files#diff-3aaaae2a7d6e9da3ba436acbc17177367fbe44281536edff80c69d399ee2d99c

Reply:
Most likely. I was worried about the blast radius